sopel/modules/reddit.py

```diff
@@ -258,14 +258,14 @@ def subreddit_info(bot, trigger, match, commanded=False):
     if match_lower in ['all', 'popular']:
         message = '{link}{nsfw} | {public_description}'
         nsfw = ' ' + bold(color('[Possible NSFW]', colors.ORANGE))
-        link = "https://reddit.com/r/" + match_lower
+        link = 'https://reddit.com/r/' + match_lower
         public_description = ''
         if match_lower == 'all':
-            public_description = ('Today\'s top content from hundreds of '
+            public_description = ("Today's top content from hundreds of "
                                   'thousands of Reddit communities.')
         elif match_lower == 'popular':
             public_description = ('The top trending content from some of '
-                                  'Reddit\'s most popular communities')
+                                  "Reddit\'s most popular communities")
```

Comment:
```suggestion
                                  "Reddit's most popular communities")
```
Does this flake plugin not catch these?

Reply:
I don't think it's designed to catch this. zheller/flake8-quotes#98

Just another reason I'm disappointed in how this turned out, and wish I'd experimented with it before adding to the wishlist so Exirel didn't waste time implementing. 😦